src/core/electric_utility.jl

```diff
@@ -160,10 +160,9 @@ struct ElectricUtility
         ### Grid Climate Emissions Inputs ### 
         # Climate Option 1 (Default): Use levelized emissions data from NREL's Cambium database by specifying the following fields:
         cambium_scenario::String = "Mid-case", # Cambium Scenario for evolution of electricity sector (see Cambium documentation for descriptions). 
-            ## Options: ["Mid-case",  "Mid-case with tax credit expiration",  "Low renewable energy cost", "Low renewable energy cost with tax credit expiration",   "High renewable energy cost", "High electrification",  "Low natrual gas prices", "High natrual gas prices", "Mid-case with 95% decarbonization by 2050",  "Mid-case with 100% decarbonization by 2035"]
```

Comment:
Why get rid of option comment?

Reply:
I got rid of it here I think just to save space, since we have the full list of options up above in the markdown text. But, I think we don't have consistency throughout the code in whether we duplicate explanatory text down here too. Do you have a strong opinion on this?